lib/pure/uri.nim

```diff
@@ -123,13 +123,13 @@ func decodeUrl*(s: string, decodePlus = true): string =
   setLen(result, j)
 
 func encodeQuery*(query: openArray[(string, string)], usePlus = true,
-    omitEq = true): string =
+    omitEq = true, sep = '&'): string =
```

Comment:
does this belong to this PR?

Reply:
no this is another PR I'm working on.

Reply:
@Zectbumo you should then only make this PR change the posix stuff and not touch the uri stuff